lib/Simulation.thy

```diff
@@ -96,7 +96,7 @@ where
 
 text \<open>
   Refinement is a global property over all executions and/or all
-  hoare triples. As this is hard to show, we define the weaker concept
+  hoare triples. As this is hard to show, we define the stronger concept
```

Comment:
I think it turns out that they are actually equivalent, so probably better to just remove the word entirely (just "we define the concept of")